group_project/public/css/group_project.css

```diff
@@ -254,6 +259,7 @@
 }
 
 .upload_form .xblock-reveal.reveal-modal.open{
+  position: fixed;
```

Comment:
Rest of the changes address misalignment of the `Browse` buttons on smaller screens. You might want to incorporate that one in your PR as well.